spk/bazarr/Makefile

```diff
@@ -11,10 +11,7 @@ DEPENDS = cross/$(SPK_NAME)
 DEPENDS += cross/unrar
 DEPENDS += cross/numpy
 DEPENDS += cross/webrtcvad
-DEPENDS += cross/gevent
-DEPENDS += cross/gevent-websocket
```

Comment:
Is `gevent` no longer needed?

Reply:
Apparently so https://github.com/morpheus65535/bazarr/releases

Reply:
thnx